main.go

```diff
@@ -80,7 +78,7 @@ func init() {
 func printVersion() {
 	setupLog.Info(fmt.Sprintf("Go Version: %s", goruntime.Version()))
 	setupLog.Info(fmt.Sprintf("Go OS/Arch: %s/%s", goruntime.GOOS, goruntime.GOARCH))
-	setupLog.Info(fmt.Sprintf("Version of operator-sdk: %v", sdkVersion.Version))
+	// setupLog.Info(fmt.Sprintf("Version of operator-sdk: %v", sdkVersion.Version))
```

Comment:
Was this intentional?

Reply:
yes that package is no longer supported, commented it out because can use an alternate package. 
Will add that.

Reply:
there is no alternate library, only way is to use os/exec - which might be an overkill for a print statement.

Reply:
Okay. Makes sense. Can we remove the line instead of commenting it?